### DISCUSSION TITLE
import css modules

### DISCUSSION THREAD

**Discussion by TbirdDuncan** [preactjs/preact-cli#1653] [Q&A] [answered]
Hi, very curious as to how to go about using css in the cli standard setup. I see that it exists in the top folder on the localhost, however the styles are not being used.

**Comment by TbirdDuncan** [comment, marked as answer]
@rschristian

**Reply by rschristian** [reply]
There's two variations of CSS use in the default Preact-CLI app: you get CSS Modules in `routes/` and `components/`, everywhere else just does a side-effectful import of stylesheets instead.

`src/components/header.jsx`
```js
import style from './style.css';
```

`src/index.js`
```js
import './style.css';
```

You can customize this using your `preact.config.js`.
